### PR TITLE
 Changes according to security warnings of MSVC

### PR DESCRIPTION
**Pull request by DoumanAsh**
Surprisingly tests doesn't work for me under MS compiler.
I'll try to work it out somehow... For now lets your CI to test for me

### CODE REVIEW

**Comment by DoumanAsh** [comment]
All tests are passed aside from crashing builder_test which asserts somewhere in VC++ std (i suspect it cannot deal with some unicode characters)

**Comment by mayah** [comment]
Sorry but I'm a bit busy today, I'll check this later.

File: include/toml/toml.h [hunk 1]
```diff
@@ -25,6 +19,12 @@
 #include <utility>
 #include <vector>
 
+#if defined(_MSC_VER)
+#undef sscanf
+//NOTE: sscanf_s requires to specify sizes of buffers to store %c or %s
+#define sscanf sscanf_s
```
Comment (mayah) [inline include/toml/toml.h]: I don't think it's good to pollute global namespace by define.

Comment (DoumanAsh) [inline include/toml/toml.h]: Yeah, you definitely right about it now i think about it.
I'll remove ifdef compeltely i think and just move usage of correct function on line 894

**Comment by DoumanAsh** [comment]
I removed re-define of `sscanf`

**Comment by mayah** [comment]
LGTM! Thanks a lot

**Pull request merged**